joomla/joomla-cms
# [4.4] Remove obsolete exclude pattern from ruleset.xml for phpcs

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) removes obsolete exclude patterns from the "ruleset.xml" file which is used by phpcs e.g. in the ci tests with Drone.

Obsolete means the corresponding file or folder has been removed (or moved to another place) with some PR but the exclude pattern has been forgotten about.

The following obsolete entries are removed:
- Line no. 237: File "plugins/content/joomla/joomla.php" has been moved to another place and been changed with PR #40561 .
- Line no. 238: File "plugins/content/loadmodule/loadmodule.php" has been moved to another place and been changed with PR #40561 .
- Line no. 239: File "plugins/content/pagebreak/pagebreak.php" has been moved to another place and been changed with PR #40561 .
- Line no. 244: File "plugins/user/joomla/joomla.php" has been moved to another place and been changed with PR #40180 .

The list of obsolete exclude patterns has been created with help of a small tool (PHP CLI script) which I provide with PR #40278 .

### Testing Instructions

Code review: Check that the removed exclude patterns belong to the files mentioned in the description and the files don't exist on a clean, current 4.4-dev branch.

Verify that the PHPCS step succeeded in the Drone ci checks for this PR here.

### Actual result BEFORE applying this Pull Request

File "ruleset.xml" in the Joomla root contains exclude patterns for files which doesn't exist anymore in the 4.4-dev branch.

### Expected result AFTER applying this Pull Request

File "ruleset.xml" in the Joomla root doesn't contain these exclude patterns.

PHPCS still succeeds in Drone for this PR.

### Link to documentations
Please select:
